src/scenarios/shared/runner.py

```diff
@@ -61,7 +92,7 @@ def parseargs(self):
         Parses input args to the script
         '''
         parser = ArgumentParser()
-        subparsers = parser.add_subparsers(title='subcommands for scenario tests', required=True, dest='testtype')
+        subparsers = parser.add_subparsers(title='subcommands for scenario tests', dest='testtype')
```

Comment:
Same question as above; why not rely on the argument parser to enforce required parameters?